test/libp2p/peer.cpp

```diff
@@ -40,6 +40,7 @@ BOOST_FIXTURE_TEST_SUITE(p2p, P2PFixture)
 
 BOOST_AUTO_TEST_CASE(host)
 {
+	cnote << "p2p/host...";
```

Comment:
are these left here by accident? It may become annoying if we spam `stdout` when runnign the tests.